packages/miniflare/src/shared/log.ts

```diff
@@ -7,11 +7,11 @@ const cwdNodeModules = path.join(cwd, "node_modules");
 
 const LEVEL_PREFIX: { [key in LogLevel]: string } = {
 	[LogLevel.NONE]: "",
-	[LogLevel.ERROR]: "err",
-	[LogLevel.WARN]: "wrn",
-	[LogLevel.INFO]: "inf",
-	[LogLevel.DEBUG]: "dbg",
-	[LogLevel.VERBOSE]: "vrb",
+	[LogLevel.ERROR]: "error",
+	[LogLevel.WARN]: "warn",
+	[LogLevel.INFO]: "info",
+	[LogLevel.DEBUG]: "debug",
+	[LogLevel.VERBOSE]: "verbose",
```

Comment:
This shortening wasn't necessary, and adding a few extra characters makes these clearer

Reply:
They could be, but they really shouldn't be. We also haven't really considered logs to be part of the semver contract for Wrangler/Miniflare in the past, so I think this should be fine?